app/components/post_list/post_list.js

```diff
@@ -145,22 +122,42 @@ export default class PostList extends PureComponent {
 
         return (
             <Post
-                highlight={post.highlight}
-                postId={post.id}
+                postId={postId}
+                previousPostId={previousPostId}
+                nextPostId={nextPostId}
+                highlight={highlightPostId && highlightPostId === postId}
                 renderReplies={renderReplies}
-                isFirstReply={post.isFirstReply}
-                isLastReply={post.isLastReply}
                 isSearchResult={isSearchResult}
                 shouldRenderReplyButton={shouldRenderReplyButton}
-                commentedOnPost={post.commentedOnPost}
                 onPress={onPostPress}
                 navigator={navigator}
             />
         );
     };
 
+    renderFooter = () => {
+        if (this.props.showLoadMore) {
+            return <LoadMorePosts theme={this.props.theme}/>;
+        } else if (this.props.channelId) {
+            // FIXME: Only show the channel intro when we are at the very start of the channel
```

Comment:
I think this is already fixed, isn't it?

Reply:
No idea. I just copied the comment from the old version when I moved it around